mobile/ios/Podfile

```diff
@@ -15,15 +15,35 @@ target 'CompanionKit' do
     # Pods for testing
   end
 
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
-  use_flipper!
+  use_flipper!({ 'Flipper-Folly' => '2.5.3', 'Flipper' => '0.87.0', 'Flipper-RSocket' => '1.3.1' })
+
+  # Fix for Folly Pod on Xcode 12.5
+
+  def find_and_replace(dir, findstr, replacestr)
```

Comment:
@MarkDavies0  just curious - was most of this auto-generated or did you write it yourself? Make sure to link the source if you're using a known fix e.g. stack overflow.

Reply:
It comes from a response to an issue on the Facebook/flipper repo which is what I linked to above. Is there a certain style for code attribution comments in the actual file or should I just add the link?

Reply:
Yeah I think just a commented link would be good